components/prism-editorconfig.js

```diff
@@ -0,0 +1,20 @@
+Prism.languages.editorconfig = {
+	// https://editorconfig-specification.readthedocs.io/en/latest/
+	'comment': /[;#].*$/m,
+	'section': {
+		pattern: /^[ \t]*\[.+]/m,
```

Comment:
```suggestion
		pattern: /^[ \t]*\[.+\]/m,
```

Reply:
My IDE highlights it as a redundant escape.
![image](https://user-images.githubusercontent.com/3684355/87204790-67ffd780-c30e-11ea-918e-c1439a1d0662.png)
Should I just ignore this?

Reply:
Fixed all comments except this one

Reply:
Technically, it is redundant. I don't like unescaped `]`a for various reasons, but this shouldn't hold this pull back.